TINY-8441: Updated docs links for 6.0 (#7657)
tinymce/tinymce: modules/tinymce/CHANGELOG.md

- The default value for the `schema` option has been changed to `html5` #TINY-8261
- The default value for the `table_style_by_css` option has been changed to `true` #TINY-8259
- The default value for the `table_use_colgroups` option has been changed to `true` #TINY-8259

> - The URL links to the TinyMCE docs reference site #TINY-8441

Reviewer: ```suggestion
```
This shouldn't be in the TinyMCE changelog because it's not something that's shipped as part of TinyMCE, instead it's part of the docs.

## 5.0.0 - 2019-02-04

> Full documentation for the version 5 features and changes is available at https://www.tiny.cloud/docs/tinymce/5/5_0_0-release-notes-overview.html

Reviewer: ```suggestion
Full documentation for the version 5 features and changes is available at https://www.tiny.cloud/docs/tinymce/5/release-notes/release-notes50.html
```
This link 404s/doesn't exist and I believe this suggestion is the correct one.

Reviewer 2: Lee walked me through this, so accepting on Mike's behalf due to time constraints.

Developer preview 1

> Initial list of features and changes is available at https://tiny.cloud/docs-preview/release-notes/new-features/

Reviewer: This shouldn't have been removed, as now there's nothing listed there.

- [ ] Revert this change

Author: its a 404 dead link

Reviewer: Removing it is still not the right solution then... we need to fix the link or add something back so that we don't have a version without any entries. I'd suggest linking to https://www.tiny.cloud/docs/release-notes/release-notes50/#newfeatures instead.

Developer preview 1

> Initial list of features and changes is available at https://tiny.cloud/tinymce/5/release-notes/release-notes50.html

Reviewer: ```suggestion
Initial list of features and changes is available at https://www.tiny.cloud/docs/tinymce/5/release-notes/release-notes50.html
```
This link is also incorrect (missing the `docs` part of the path).

Reviewer 2: Lee walked me through this, so accepting on Mike's behalf due to time constraints.
